### ISSUE TITLE
Missing support for Windows ARM (aarch64)

### ISSUE THREAD

**Issue by rock3r** [JetBrains/skiko#641]
There is no binary published for Windows ARM; this means that when running a Compose for Desktop app (after forcing the `windows_x64` dependency in Gradle), you get this error:

```
Exception in thread "main" org.jetbrains.skiko.LibraryLoadException: Cannot find skiko-windows-arm64.dll.sha256, proper native dependency missing.
	at org.jetbrains.skiko.Library.findAndLoad(Library.kt:105)
	at org.jetbrains.skiko.Library.load(Library.kt:59)
	at org.jetbrains.skia.impl.Library$Companion.staticLoad(Library.jvm.kt:12)
	at androidx.compose.ui.ConfigureSwingGlobalsForCompose_desktopKt.configureSwingGlobalsForCompose(ConfigureSwingGlobalsForCompose.desktop.kt:49)
	at androidx.compose.ui.ConfigureSwingGlobalsForCompose_desktopKt.configureSwingGlobalsForCompose$default(ConfigureSwingGlobalsForCompose.desktop.kt:38)
	at androidx.compose.ui.window.Application_desktopKt.application(Application.desktop.kt:110)
	at androidx.compose.ui.window.Window_desktopKt.singleWindowApplication(Window.desktop.kt:304)
	at androidx.compose.ui.window.Window_desktopKt.singleWindowApplication$default(Window.desktop.kt:289)
	at org.jetbrains.jewel.samples.standalone.MainKt.main(Main.kt:12)
	at org.jetbrains.jewel.samples.standalone.MainKt.main(Main.kt)
```

This is using Compose Desktop 1.2.1.

cc @igordmn

**Comment by VladRassokhin** [comment]
Duplicate of #607

**Comment by VladRassokhin** [comment]
Probably you need to update skiko version in your project. `org.jetbrains.skiko:skiko-awt-runtime-windows-arm64` is [available](https://mvnrepository.com/artifact/org.jetbrains.skiko/skiko-awt-runtime-windows-arm64) starting from version 0.7.39.

**Comment by rock3r** [comment]
I'll double check, but I'm using skiko indirectly through Compose Multiplatform, so not sure what the effect would be

**Comment by igordmn** [comment]
Skiko with Windows ARM support is only available in Compose >= 1.3.0-...

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/SKIKO-641">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
